File: src/main/java/ux/panel/AlignVertical.java
package ux.panel;

import processing.core.PApplet;

public enum AlignVertical {
    TOP, CENTER, BOTTOM;

    public int toPApplet() {
        return switch (this) {
            case TOP -> PApplet.TOP;
            case CENTER -> PApplet.CENTER;
            case BOTTOM -> PApplet.BOTTOM;
        };
    }
}

File: src/main/kotlin/patterning/panel/AlignVertical.kt
package patterning.panel

import processing.core.PApplet

enum class AlignVertical {
    TOP,
    CENTER,
    BOTTOM;

    fun toPApplet(): Int {
        return when (this) {
            TOP -> PApplet.TOP
            CENTER -> PApplet.CENTER
            BOTTOM -> PApplet.BOTTOM
        }
    }
}